FinanceTracker/FinanceTracker/Commands/GetRecordsByCategory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FinanceTracker.Interfaces;
+using FinanceTracker.Models;
+using FinanceTracker.Utils;
+
+namespace FinanceTracker.Commands
+{
+    public class GetRecordsByCategory : ICommand
```

Comment:
Maybe its time to add interface for Commands that returns results? Like ICommand<T> with method T Execute<T>